components/resources/wallet_strings.grdp

```diff
@@ -270,6 +270,7 @@
   <message name="IDS_BRAVE_WALLET_CREATE_ACCOUNT_CARDANO_DESCRIPTION" desc="Add account modal create account Cardano description">Store Cardano assets</message>
   <message name="IDS_BRAVE_WALLET_CREATE_ACCOUNT_CARDANO_TESTNET_DESCRIPTION" desc="Add account modal create account Cardano Testnet description">Store Cardano Testnet assets</message>
   <message name="IDS_BRAVE_WALLET_CREATE_ACCOUNT_POLKADOT_DESCRIPTION" desc="Add account modal create account Polkadot description">Supports Substrate assets on Polkadot relay chain and Substrate parachains</message>
+  <message name="IDS_BRAVE_WALLET_CREATE_ACCOUNT_POLKADOT_TESTNET_DESCRIPTION" desc="Add account modal create account Polkadot Tesnet description">Supports Substrate assets on Westend relay chain and Substrate parachains</message>
```

Comment:
Should that be only "Westend relay chain"?
I assume for testnet accounts we should support only one network